### DISCUSSION TITLE
App Store Connect Internal Group Testing Release Settings

### DISCUSSION THREAD

**Discussion by bassrock** [fastlane/fastlane#20640] [Q&A]
I am currently using:

```
    #upload to testflight for our internal apps store connect nightly builds.
    upload_to_testflight(
      skip_waiting_for_build_processing: false,
      groups: ['Nightly Internal Builds'],
      submit_beta_review: false,
      ipa: BITRISE_IPA_PATH
    )
```

To enable nightly builds to an Internal App Store Connect testing group. However, it seems that Pilot recognizes the build is in the list and stays at `[00:41:07]: Waiting for App Store Connect to finish processing the new build` The build does show up in iTunes Connect and does not indicate it is processing there and I can manually move into the Internal Testing group.

Are these the correct settings for distributing to Internal Testing groups in Fastlane?

**Comment by bassrock** [comment]
I created https://github.com/fastlane/fastlane/pull/20644 which seems to solve this.

**Reply by SSheldon** [reply]
Looks like other folks are running into this issue too and #20645 was opened as an issue for it.
